### DISCUSSION TITLE
LilyGo TDisplay RP2040 needs PWR_EN pin

### DISCUSSION THREAD

**Discussion by Juanvelascoibarz** [Bodmer/TFT_eSPI#1638] [Raspberry Pi Pico]
I was no way to have something in screen with this board.
After struggling with it I found there wasn't 3.3v. at VDD ST7789 terminal.
So, following schematic I see it needs PWR_EN signal from RP2040 (GPIO22).
Then I try putting it to HIGH in arduino ide scketch example, this way backlight and screen can be active and works ok.
Someone having this behaviour with this board? . GPIO22 must be HIGH by default ?
I don't understand very well this behaviour.
![image](https://user-images.githubusercontent.com/38695758/153947332-57755fa0-275a-4a0b-8bf0-65847f2d9402.png)
![image](https://user-images.githubusercontent.com/38695758/153947480-4ff83245-cf83-406f-8ec8-f8851550dfe3.png)
![image](https://user-images.githubusercontent.com/38695758/153947965-0c8efc61-f58f-4689-ae3c-50bbcd4fd6a2.png)

Thank you for your help.

**Comment by Bodmer** [comment]
Yes, as you found out, that board has a power switch inline with the display power pin.

You have taken the correct approach of driving that pin high in the sketch.

I expect the switch has been added to allow the user to reduce power,  when running on a battery, by allowing the display to be powered down.

**Reply by Juanvelascoibarz** [reply]
Good suggestion, solved my doubt.
Thank you very much!!
